docs/app/docs/themes/page.mdx

```diff
@@ -544,3 +544,13 @@ Widget build(BuildContext context) {
   );
 }
 ```
+
+### CLI
+
+For more fine-grained control, you can use the CLI to generate a copy of the `toApproximateMaterialTheme()` inside your
+project. The generated copy can then be directly modified to fit your design needs.
+
+To generate a copy of `toApproximateMaterialTheme()`, run:
+```shell copy
+dart run forui snippet create material-mapping
+```
```

Comment:
I think this should be reflected under the main cli page. And we can probably just have a one liner hyerlinking it to there.